stake-pool/program/src/entrypoint.rs

```diff
@@ -24,3 +25,18 @@ fn process_instruction(
         Ok(())
     }
 }
+
+security_txt! {
+    // Required fields
+    name: "SPL Stake Pool",
+    project_url: "https://spl.solana.com/stake-pool",
+    contacts: "link:https://github.com/solana-labs/solana-program-library/security/advisories/new,mailto:security@solana.com,discord:https://discord.gg/solana",
```

Comment:
```suggestion
    contacts: "link:https://github.com/solana-labs/solana-program-library/security/advisories/new,mailto:security@solana.com,discord:https://solana.com/discord",
```

missed this on the other one. we don't control the discord.gg domain. retaining it requires playing a stupid game and it can be hijacked if we don't win.

Reply:
Ah ok, I'll fix that on the other one too. Thanks for pointing it out